Source/Fuse.ImageTools/Tests/ImageTools.Test.uno

```diff
@@ -10,49 +10,31 @@ namespace Fuse.ImageTools.Test
 	{
 		private Object _image = null;
 
-
 		[Test]
+		[Ignore("Only supported on dotnet", "!DOTNET")]
```

Comment:
Do we run Uno tests on Android and iOS too? I thought it was only DotNet + Native?